spec/inputs/eventlog_spec.rb

```diff
@@ -1,7 +1,11 @@
 require "logstash/devutils/rspec/spec_helper"
-require 'logstash/inputs/eventlog'
 
-describe LogStash::Inputs::EventLog, :windows => true do
+describe "LogStash::Inputs::EventLog", :windows => true do
+
+  before(:all) do
+    require 'logstash/inputs/eventlog'
```

Comment:
for now this makes the work, but I wonder if we should also improve the way we handle require libraries inside the plugin. there is not just this plugin, but also in the zeromq there is one requirement making everything "funny". what do you think?